msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -919,6 +919,8 @@ The `extensions.runtimes` property configures the sets of runtimes and actions t
 |`actions.displayName`| String | 64 characters | | Specifies the display name of the action and it isn't the label of a button or a menu item that invokes the action (which is configured with `tabs.groups.controls.label`).|
 |`actions.pinnable`| Boolean | | | Specifies that a task pane supports pinning, which keeps the task pane open when the user changes the selection. <br>Default value: `false`|
 |`actions.view`| String | 64 characters | | Specifies the view where the page must be opened. It's used only when `actions.type` is `openPage`. |
+|`actions.multiselect`| Boolean | | | Specifies whether the end user can select multiple items, such as multiple email messages, and apply the action to all of them.|
+|`actions.supportsNoItemContext`| Boolean | | | Allows task pane add-ins to activate without the Reading Pane enabled or a message selected. |
```

Comment:
Hi @Rick-Kirkham, I have a question that why requirement property is not listed in extensions.runtimes here?

Reply:
Another good catch. Fixed. But see my email with questions so that we can have better descriptions of the subproperties.